tools/harness-automation/autothreadharness/harness_case.py

```diff
@@ -315,13 +323,14 @@ def setUp(self):
         logger.info('Deleting all .pdf')
         os.system('del /q "%HOMEDRIVE%%HOMEPATH%\\Downloads\\NewPdf_*.pdf"')
         logger.info('Deleting all .xlsx')
-        os.system('del /q "%HOMEDRIVE%%HOMEPATH%\\Downloads\\ExcelReport*.xlsx"')
+        os.system('del /q "%HOMEDRIVE%%HOMEPATH%\\Downloads\\*_Excel_Report.xlsx"')
```

Comment:
I think we should add a rule instead of replacing the old one, because someone may use this tool to drive old harness release?

Reply:
Agree.  
The new directory or file name changes only exist in newer TH member release, maybe add some variable to track the Test Harness version for better backward compatibility with public release?